include/revng/PipelineC/Prototypes.h

```diff
@@ -24,15 +24,6 @@
  * nullptr can be passed instead and errors will be silently ignored.
  */
 
-/**
- * Allow setting a custom abort hook.
- * This will be called just before revng_abort() in case of an assertion
- * failure. Useful if calling from a non-C language to print extra debug
- * information.
- */
-typedef void (*AbortHook)(void);
-void rp_set_custom_abort_hook(AbortHook Hook);
```

Comment:
Nice!
So now we no longer generate trampolines which require to `+x` some memory page? Right?
Too bad we'll probably need this again once we have progress notifications callbacks.

Reply:
Yes, for now no more `+x` memory pages